Dijkstra Algorithm/README.md

```diff
@@ -0,0 +1,23 @@
+# Dijkstra-algorithm
+WWDC 2017 Scholarship Project
+Created by [Taras Nikulin](https://github.com/crabman448)
+
+## About
+Dijkstra's algorithm is used for finding shortest paths from start vertex in graph(with weighted vertices) to all other vertices. 
+More algorithm's description can be found in playground or on wikipedia:
```

Comment:
Dijkstra's algorithm is one of the most famous graph algorithms. Instead of giving a link to wikipedia, we'll want a bigger sample of what this algorithm is about. 

Some things to talk about:

- Why it is so often studied
- Real life problems that it solves
- A summary of how the algorithm works